tests/Fakes/TestResponseFactory.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors\Tests\Fakes;
+
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\ResponseInterface;
+
+class TestResponseFactory implements ResponseFactoryInterface
+{
+    public function createResponse(int $code = 200, string $reasonPhrase = ''): ResponseInterface
```

Comment:
Maybe we can use constants for the status codes?
Symfony (https://symfony.com/doc/current/components/http_foundation.html) f.e. provides the HTTP status codes or maybe you know something else that we could use for that?

Reply:
I don't see any advantage for the overhead to use constants for the response codes. Why do you prefer constants?

Reply:
For the tests it would be fine like it is, but if we need them later in the code I would prefer constants, because it's more readable and less error prone. 🤔